ansys-grantami-serverapi-openapi/src/ansys/grantami/serverapi_openapi/models/microsoft_asp_net_core_mvc_formatters_i_output_formatter.py

```diff
@@ -0,0 +1,91 @@
+"""
```

Comment:
I don't think these are real models. Are they referenced by any APIs?

Reply:
This one doesn't seem to be. The next one is a response for a license check endpoint